File: Python/program27.py
def main():
    
    iValue = int(input("Enter number \n"))
    
    iRet = Factorial(iValue)
    
    print("Factorial  is ", iRet)
    
def Factorial(iNo):
    Fact = 1
    if(iNo < 0):
        print("Invalid Input\n")
        return -1
    
    for iCnt in range(1,iNo + 1):
        Fact = Fact * iCnt
    return Fact
    

    
    
if __name__ == "__main__":
    main()

File: Java/program27.java
//Imput : 5
//Output : 1*2*3*4*5 = 120

import java.util.*;

class Factorial
{
    public int Fact(int iNo)
    {
        if(iNo < 0)
        {
            return -1;
        }
        int iCnt = 0,iFact = 1;

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            iFact = iFact * iCnt;
        }

        return iFact;
        
    } 
}

class program27
{
    public static void main(String A[])
    {
        int iValue = 0;
        int iRet = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number :");
        iValue = sobj.nextInt();

        Factorial fobj = new Factorial();
        iRet = fobj.Fact(iValue);

        if(iRet == -1)
        {
           System.out.println("Invlaid Input"); 
        }
        else
        {
            System.out.println("Factorial of number  is :"+iRet);

        }

               
        
        
    }
}